src/site/_includes/demo.njk

```diff
@@ -6,13 +6,25 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>{{ title }}</title>
+  <style>
+    html, body {
+      height: 100%;
+    }
+    figure {
```

Comment:
I appreciate the sensible defaults here. But in this literal demo, the image actually ends up being too big for the patterns preview thing.

Can you add CSS for e.g. `figure > img` so it has `max-width: 100%` and `object-fit: contain`? (There might be more to make it fit in the center, that's just off the top of my head).